djangocms_versioning/models.py

```diff
@@ -37,6 +37,16 @@ def save(self, **kwargs):
             for version in to_archive:
                 version.archive(self.created_by)
                 version.save()
+        # Only one published version is allowed per grouper. Set all other
```

Comment:
Shouldn't this happen in `Version.publish` method? This way you can pass correct user to `version.unpublish`, as currently it's the version author, not the user who published it.